src/components/donation/Steps/Summary/Summary.tsx

```diff
@@ -40,15 +35,20 @@ export default function Summary({
       amount={amount}
       splitLiq={liquidSplitPct}
       onBack={() =>
-        recipient.hide_bg_tip
-          ? dispatch(setStep("splits"))
-          : dispatch(setStep("tip"))
+        setState({
+          ...props,
+          step: init.recipient.hide_bg_tip
```

Comment:
example of PR description point `B.` `Summary` knows well where state would end up if user clicks `back`